src/components/StatusDetails/StatusDetails.tsx

```diff
@@ -32,42 +36,33 @@ const findCurrentStop = (stops: Stop[]) => {
   }
 };
 
-const getMinutesUntil = (target: Date) => {
-  return Math.ceil((target.getTime() - Date.now()) / 1000 / 60);
-};
-
 const StatusDetails = ({ status }: StatusDetailsProps) => {
+  const [nextStop, setNextStop] = useState<Stop>();
   const { stops: allStops } = useStops(
-    status?.train.hafasId ?? '',
-    status?.train.lineName ?? '',
-    status?.train.origin.departurePlanned ?? '',
-    status?.train.origin.id.toString() ?? ''
+    status.train.hafasId ?? '',
+    status.train.lineName ?? '',
+    status.train.origin.departurePlanned ?? '',
+    status.train.origin.id.toString() ?? ''
   );
 
+  const direction = allStops.at(-1)?.name;
```

Comment:
Das ist leider keine gute Idee, die beim Aufruf der Seite ist das dann erstmal undefined obwohl wir die Daten ja in `status.train.destination.name` schon in der ersten Hydration haben

Reply:
Ja, nur leider sind das unterschiedliche Stationen. Wir sollten wahrscheinlich die Stops auch einmal auf dem Server laden und dann die Direction als Prop mit an die Page übergeben.

(Noch besser wäre natürlich direkt HAFAS zu callen nach der tatsächlichen Direction statt dem Fallback auf den letzten Stop, aber dann bekommen wir serverseitig evtl. ein Rate-Limit Problem)